cogsmisc/customization.py

```diff
@@ -648,6 +642,41 @@ async def gvar_list(self, ctx):
         for m in say_list[1:]:
             await ctx.send(m)
 
+    # FIXME: temporary commands to aid testers with lack of dashboard
```

Comment:
There is an ongoing conversation in Discord on whether or not to keep these gvar commands - for prod release, they're commented out.